Xamarin.Forms.Core/ScrollView.cs

```diff
@@ -228,6 +232,10 @@ protected override SizeRequest OnSizeRequest(double widthConstraint, double heig
 					widthConstraint = double.PositiveInfinity;
 					heightConstraint = double.PositiveInfinity;
 					break;
+				case ScrollOrientation.Neither:
```

Comment:
Can we add a unit test ?

Reply:
Wow, I think that would be great

Reply:
@rmarinho I've started updating unit-test in a9a89f848b32dc899251f8f0be9ac9dc11279874, but I feel a little bit lost. May you assist me in what test cases should I cover?

Reply:
need to test the size is the same with Neither, and maybe that the scrolled even't doesn't fire?

Reply:
@rmarinho  Thank you for recomendations! I've added tests'n'fixed in 0128f0033c7593222477e661b69d0fa7d96f5549 and 445bc9208d87b3dcd7657e28cc6f12126d113926. Btw, that helped to fixe firing scroll event when its not necessary!